enterprise/config/BUILD

```diff
@@ -10,7 +10,10 @@ filegroup(
             "buildbuddy.release.yaml",
             "executor.release.yaml",
         ],
-        "//conditions:default": glob(["**"]),
+        "//conditions:default": glob(["*.yaml"]) + [
+            "//enterprise/config/test:buildbuddy.noauth.yaml",
+            "//enterprise/config/test:buildbuddy.selfauth.yaml",
+        ],
```

Comment:
nit: I think we should probably have the tests which need these configs explicitly depend on them, instead of including them in the config_files target. It feels a little weird to include test configs in a filegroup target which has other non-test files as well.

Reply:
tbh, I don't fully understand the BUILD file logic in this `enterprise/config` dir.

It seems like we could remove `:bundle` entirely since, as well as change the `filegroup `into `exports_files` and force other targets to explicitly depend on the right target.

So let's do this instead: I will remove the test config files from the filegroup in this PR. And I will investigate removing the filegroup and `:bundle` in a separate PR. WDYT?

Reply:
Sounds good, although I don't think removing the filegroup or bundle target is needed so just removing the test config files seems good enough for now.

Reply:
SGTM